data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals/heal.cfg

```diff
@@ -1,3 +1,5 @@
+# wmllint: no translatables
```

Comment:
Does it make much sense to run wmllint on the test files at all?

Reply:
I don't know, given we don't run wmllint on anything in mainline currently. @stevecotton said it should be added, which is why it's there.

Reply:
I thought we're aiming for (eventually, but surely not within 2024) running wmllint on mainline. #7082

Reply:
Running wmllint on mainline is one thing but I’m wondering about the tests specifically.

Reply:
At least for tests that are specifically testing deprecated/removed things, it would be incorrect.